### ISSUE TITLE
About ADE dataloader

### ISSUE THREAD

**Issue by YininKorea** [yassouali/pytorch_segmentation#84]
Why should we change the value of labels of ADE dataset to [-1,149]?
